docs/content/dataset_standards.md

```diff
@@ -177,10 +177,233 @@ The stepping data inside the episode group is divided into some required `datase
+                        <li class="dataset">additional_datasets</li>
+                    </ul>
+                </li>
+            </ul>
+        </li>
+        <li class="folder-closed">episode_1</li>
+        <li class="folder-closed">episode_2</li>
+        <ul><br></ul>
+        <li class="folder-closed">episode_id</li>
+    </ul>
+</li>
+</ul>
+</div>
+
+<div class="only-dark">
+<ul class="directory-list">
+    <li class="file-white" style="color:white">main_data.hdf5
+        <ul class="white">
+            <li class="folder-white">episode_0
+                <ul class="white">
+                    <li class="dataset-white">observations</li>
+                    <li class="folder-white">actions
+                        <ul>
+                            <li class="dataset-white">_index_0</li>
+                            <li class="folder-white">_index_1_
+                            <ul>
+                            <li class="dataset-white"> _index_0 </li>
+                            <li class="dataset-white"> _index_1_ </li>
+                            </ul>
+                            </li>
+                        </ul>
+                    </li>
+                    <li class="dataset-white">terminations</li>
+                    <li class="dataset-white">truncations</li>
+                    <li class="dataset-white">rewards</li>
+                    <li class="folder-white">infos
+                    <ul class="white">
+                        <li class="dataset-white">infos_datasets</li>
+                        <li class="folder-white">infos_subgroup
+                        <ul class="white">
+                            <li class="dataset-white">more_datasets</li>
+                        </ul>
+                        </li>
+                    </ul>
+                    </li>
+                    <li class="folder-white">additional_groups
+                        <ul class="white">
+                            <li class="dataset-white">additional_datasets</li>
+                        </ul>
+                    </li>
+                </ul>
+            </li>
+            <li class="folder-white-closed">episode_1</li>
+            <li class="folder-white-closed">episode_2</li>
+            <ul class="white"><br></ul>
+            <li class="folder-white-closed">episode_id</li>
+        </ul>
+    </li>
+</ul>
+</div>
+
+Note how the `Tuple` space elements are assigned corresponding keys of the format `f"_index_{i}"` were `i` is their index in the `Tuple` space.
+
+
+The required `datasets` found in the episode groups correspond to the data involved in every Gymnasium step call `obs, rew, terminated, truncated, info = env.step(action)`: `observations`, `actions`, `rewards`, `terminations`, and `truncations`. These datasets are `np.ndarray` or nested groups of `np.ndarray` and other groups, depending on the observation and action spaces, and the space of all datasets under each required top-level episode key is equal to:
```

Comment:
Put a colon : before listing the quoted code for obs, reward etc

Reply:
added